src/ripper/ripcddialog.cpp

```diff
@@ -211,15 +213,24 @@ void RipCDDialog::ClickedRipButton() {
 
   // create and connect Ripper instance for this task
   Ripper* ripper = new Ripper(cdda_device_->song_count(), this);
+  if (!ripper) {
```

Comment:
I don't think this can ever happen. If enabled, the assert in the Ripper constructor would call abort.

Reply:
You are of course right, for some stupid reason I was scared of new returning null, but that can't happen... Removed the check now.